frontend/appflowy_flutter/lib/mobile/presentation/home/favorite_folder/mobile_home_favorite_folder_header.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/mobile/presentation/home/turn_box.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flutter/material.dart';
+
+class MobileFavoriteFolderHeader extends StatefulWidget {
+  const MobileFavoriteFolderHeader({
+    super.key,
+    required this.onPressed,
+    required this.onAdded,
+    required this.isExpanded,
+  });
+
+  final VoidCallback onPressed;
+  final VoidCallback onAdded;
+  final bool isExpanded;
+
+  @override
+  State<MobileFavoriteFolderHeader> createState() =>
+      _MobileFavoriteFolderHeaderState();
+}
+
+class _MobileFavoriteFolderHeaderState
+    extends State<MobileFavoriteFolderHeader> {
+  double _turns = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        Expanded(
+          child: FlowyButton(
+            text: FlowyText.semibold(
+              LocaleKeys.sideBar_favorites.tr(),
+              fontSize: 20.0,
+            ),
+            margin: const EdgeInsets.symmetric(vertical: 8),
+            expandText: false,
+            mainAxisAlignment: MainAxisAlignment.start,
+            rightIcon: TurnBox(
```

Comment:
Not sure what this widget is, but this looks like a candidate for https://api.flutter.dev/flutter/widgets/AnimatedRotation-class.html